action.yml

```diff
@@ -22,6 +22,10 @@ inputs:
     description: 'Fixed hostname to use.'
     required: false
     default: ''
+  distro:
```

Comment:
I'd recommend naming this `arch` as `distro` more often refers to Debian vs Fedora vs Mandrake etc.

Reply:
Sure thing, I was going by the naming here: https://pkgs.tailscale.com/stable/#static, happy to swap it.